acceptance/analyzer_test.go

```diff
@@ -643,6 +678,8 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 		when("registry case", func() {
 			it("writes analyzed.toml", func() {
+				h.SkipIf(t, runtime.GOOS == "windows", "Not yet working on Windows")
```

Comment:
Could this be removed?

Reply:
This conversation has been marked as resolved although there was no comment or the code wasn't removed.
Did I miss something?

Reply:
Ah, i thought I got them all. Let's see if i can remove it 😬

Reply:
I think that there are 5 tests that we're currently skipping on "windows". Why? Can we make them work on "windows"?

Reply:
Now I see that next to 3 of them we have a comment that they are "not relevant on Windows".
Can we expand these comments so they'll be clearer?

Reply:
All skips have been removed that I see and they all pass 🎉

Reply:
I know that it's not specifically related to your PR, but since you're already touching this file, do you mind rewriting the comments about those tests that are "Not relevant on Windows"?
We can totally do it in another PR if you prefer!

Reply:
I think we should do this in another PR

Reply:
Sounds good. Thanks!